modules/install-sno-requirements-for-installing-single-node-openshift.adoc

```diff
@@ -0,0 +1,34 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-preparing-to-install-sno.adoc
+
+[id="requirements-for-installing-single-node-openshift_{context}"]
+= Requirements for installing single node OpenShift
```

Comment:
Maybe "Requirements for installing on a single node"

No "single node OpenShift" anywhere.  I won't mark other instances.  If you want a suggested replacement, please let me know.